enterprise/libs.amazon/nbproject/project.xml

```diff
@@ -39,23 +39,66 @@
             <friend-packages>
                 <friend>org.netbeans.modules.cloud.amazon</friend>
                 <package>com.amazonaws</package>
+                <package>com.amazonaws.auth</package>
+                <package>com.amazonaws.client</package>
+                <package>com.amazonaws.client.builder</package>
                 <package>com.amazonaws.services.elasticbeanstalk</package>
                 <package>com.amazonaws.services.elasticbeanstalk.model</package>
                 <package>com.amazonaws.services.s3</package>
+                <package>com.amazonaws.services.s3.internal</package>
```

Comment:
I packages are already called `internal` I would not expose them (if they are needed, then ignore this :-))

Reply:
com.amazonaws.services.s3.internal needs to be exposed or building the Amazon EB module will fail due to not being able to access com.amazonaws.services.s3.internal.S3DirectSpi

Reply:
Why? they should be in the same module and thus being able to directly access all classes.

Reply:
All I can say right now is that the `AmazonS3` interface (in `com.amazonaws.services.s3` package) extends the `com.amazonaws.services.s3.internal.S3DirectSpi` interface, and if the s3.internal pacakge is not exposed building the EB module will fail with:

```
 error: cannot access S3DirectSpi
            s3.getObjectMetadata(bucket, DEFAULT_EMPTY_APPLICATION);
  class file for com.amazonaws.services.s3.internal.S3DirectSpi not found
```

I might do some testing tomorrow to see if this always happens in NB modules when an Interface extends another Interface in an inacessible package... but it does seem a bit odd.

Reply:
I reproduced it - yeah the resolver needs access to the internal class. Thanks for checking.